css/style.css

```diff
@@ -254,10 +254,14 @@ button:hover {
 	font-size: 12pt;
 	font-weight: 700;
 	border: 0;
-	background-color: transparent;
+	background-color: white;
 	min-height: initial;
 }
 
+.stack form {
+	width: 100%;
+}
+
 .stack h2 button,
 .stack .stack-actions {
```

Comment:
the margin should ideally only apply to the buttons, not to the container. otherwise the spacing is a bit to big.